analysis_options.yaml

```diff
@@ -0,0 +1,34 @@
+analyzer:
+  strong-mode: true
+  errors:
+    unused_import: error
+    unused_local_variable: error
+    dead_code: error
+linter:
+  rules:
```

Comment:
Lints give me hives; they aren't well-curated or -maintained in general. If I were very confident in an individual lint I'd be okay with enabling it, but this scattershot approach makes me worried that we'll end up chasing down false positives and bugs.